commit 8231e6ff5d108787048b225ec9a06d7056f2e3e3 (refs/changes/35/2635/2)
Author: Angela Fong <angela.fong@joyent.com>
Date:   2017-09-22T15:56:41+00:00 (2 years, 1 month ago)
    
    ADMINUI-2387 Networking: Read-only operators should not get the Delete buttons

diff --git a/www/js/components/pages/networking/fabric-vlan-list.jsx b/www/js/components/pages/networking/fabric-vlan-list.jsx
index 95ce5fb0..800c4453 100644
--- a/www/js/components/pages/networking/fabric-vlan-list.jsx
+++ b/www/js/components/pages/networking/fabric-vlan-list.jsx
@@ -75,7 +75,7 @@ var FabricVlansListItem = React.createClass({
                     <strong>Description</strong>
                     <span className="value">{data.description}</span>
                 </div>
-                {adminui.user.role('operators') || adminui.user.role('readers') ?
+                {adminui.user.role('operators') ?
                     <div className="actions">
                         <a onClick={this.handleEdit} className="edit"><i className="fa fa-pencil"></i> Edit</a>
                         <a onClick={this.deleteVlan} className="delete"><i className="fa fa-trash-o"></i> Delete</a>
diff --git a/www/js/components/pages/networking/networks-list.jsx b/www/js/components/pages/networking/networks-list.jsx
index fe776cc6..fe7c466e 100644
--- a/www/js/components/pages/networking/networks-list.jsx
+++ b/www/js/components/pages/networking/networks-list.jsx
@@ -191,7 +191,7 @@ var NetworksList = React.createClass({
                 <strong>MTU / NIC Tag MTU</strong>
                 <span className="value">{network.mtu} / {this.nicTagsMtu[network.nic_tag]}</span>
             </div>
-            {this.props.showActions && adminui.user.role('operators') || adminui.user.role('readers') ?
+            {this.props.showActions && adminui.user.role('operators') ?
                 (<div className="actions">
                     <a onClick={this.deleteNetwork.bind(this, model)} className="delete"><i className="fa fa-trash-o"></i> Delete</a>
                 </div>)
